src/Core/Grid/Definition/Factory/CategoryGridDefinitionFactory.php

```diff
@@ -233,6 +235,11 @@ protected function getFilters()
                         'data-url' => $this->resetActionUrl,
                         'data-redirect' => $this->redirectActionUrl,
                     ],
+                    'reset_route' => 'admin_common_reset_search_by_filter_id',
```

Comment:
Mmmm, I think you should remove the `attr` part

Reply:
done